src/app/models/workspace.server.model.ts

```diff
@@ -25,7 +25,7 @@ const compileWorkspaceModel = (db: Mongoose): IWorkspaceModel => {
         ],
         required: 'Workspace title cannot be blank',
         minlength: [4, 'Workspace title must be at least 4 characters'],
-        maxlength: [50, 'Workspace title can have a maximum of 50 characters'],
+        maxlength: [25, 'Workspace title can have a maximum of 30 characters'],
```

Comment:
just wondering if we can reuse the constants in `workspaceValidation`, to avoid case where we forget to change one

Reply:
[f9c3c65](https://github.com/opengovsg/FormSG/pull/6836/commits/f9c3c6501b6c1b737a0a84c2f406ca840823961f)